src/assets/base.css

```diff
@@ -6,3 +6,28 @@
 body {
   height: 100vh;
 }
+
+canvas { 
+  display: block;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0; 
+  left: 0; 
+  animation-name: fadeinout;
+  animation-duration: 6s;
```

Comment:
We will work on the transition next. 
I'm not sure what this does.... changing it doesn't seem to affect the frontend.